config/runtime/tests/kselftest.jinja2

```diff
@@ -25,3 +25,4 @@
         TST_CMDFILES: '{{ collections }}'
         TST_CASENAME: '{{ tests }}'
         SKIP_INSTALL: True
+	ENV: '{{ env }}'
```

Comment:
```suggestion
        ENV: '{{ env }}'
```

No tabs please, see the error message from https://staging.kernelci.org:9000/viewer?node_id=66c5ade523f314e3b22199ff

Reply:
@a-wai fixed it